db/schema.rb

```diff
@@ -425,22 +425,17 @@
   end
 
   create_table "bank_accounts", force: :cascade do |t|
-    t.string "_routing_number"
     t.integer "account_type"
     t.datetime "created_at", null: false
     t.string "encrypted_account_number"
     t.string "encrypted_account_number_iv"
     t.string "encrypted_bank_name"
     t.string "encrypted_bank_name_iv"
-    t.string "encrypted_routing_number"
-    t.string "encrypted_routing_number_iv"
     t.string "hashed_account_number"
-    t.string "hashed_routing_number"
     t.bigint "intake_id"
     t.string "routing_number"
     t.datetime "updated_at", null: false
     t.index ["hashed_account_number"], name: "index_bank_accounts_on_hashed_account_number"
-    t.index ["hashed_routing_number"], name: "index_bank_accounts_on_hashed_routing_number"
```

Comment:
We're dropping an index here. It looks like we didn't add a similar index for `routing_number`. Might be good to add the `routing_number` index.

Reply:
good call! added

Reply:
good call! i added the index